deploy-service/common/src/main/java/com/pinterest/deployservice/bean/EnvironBean.java

```diff
@@ -540,6 +544,14 @@ public void setStage_type(EnvType stage_type) {
         this.stage_type = stage_type;
     }
 
+    public Boolean getSOX() {
+        return is_sox;
+    }
+
+    public void setSOX(Boolean is_sox) {
```

Comment:
I think getters/setters needs to match property name.

Reply:
Which part of the getter/setter? I'm not seeing what is not matching.

Reply:
If property name is is_sox then getIs_sox()/setIs_sox().